sys/include/sema.h

```diff
@@ -58,92 +78,76 @@ typedef struct {
  * @param[in] value Initial value for the semaphore.
  *
  * @return  0 on success.
- * @return  -EINVAL, if semaphore is invalid.
```

Comment:
> About this you already complained earlier (here [1] and here [2]). 

Mh, I would argue that [1] was about the change of the value itself and [2] was about the undocumented precondition (which is a different scope than return values because an unmet precondition leads to a crash, while an error reported by a return-value can be handled gracefully). However the comment in OP of this conversation was about arguing to remove, because according to this documentation the function only has one possible return value (`0`) and will always succeed, so having a return value seems redundant.